website_supplier_list/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# © 2016 Michael Viriyananda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    website_supplier_published = fields.Boolean(
```

Comment:
I'd  call this `website_published`, so in v9 it can benefit from just inheriting [`website.published.mixin`](https://github.com/odoo/odoo/blob/7682760dcad232829fbb5d4221f66a1ebb9d6c91/addons/website/models/website.py#L731).